packages/desktop-client/src/components/spreadsheet/CellValue.tsx

```diff
@@ -15,16 +15,18 @@ import { type Binding } from '.';
 
 type CellValueProps = {
   binding: string | Binding;
+  goalStatus?: boolean;
```

Comment:
I would prefer keeping the CellValue agnostic of the goal status and just aware of the value returned by the binding. Since the goal status is only used here for styling, maybe the caller of this component should just define a getStyle prop?

Reply:
I pulled out the changes in this file to layer above, BalanceWithCarryover.tsx.  Its just a straight style instead of using the getStyle prop.  Is that a better option?

Reply:
I think it's still better to use getStyle so that you don't need to do another query of the balance binding

Reply:
Is there a clean way to have a getStyle prop that takes in two arguments? Thats the problem I have been trying to get around.